.github/workflows/shiny-pr-2-deploy.yaml

```
@@ -37,6 +37,9 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ github.event.pull_request.head.sha }}
+          # This job only runs for fork PRs that a maintainer has explicitly
+          # labeled DEPLOY (see the `if:` above), not arbitrary forks
+          allow-unsafe-pr-checkout: true
```

actionlint (1.7.12)

[error] input "allow-unsafe-pr-checkout" is not defined in action "actions/checkout@v4". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)